server/infra/messaging/rabbit-mq-messaging.js

```diff
@@ -1,9 +1,10 @@
 const Broker = require('rascal').BrokerAsPromised
 const config = require('./config').config
+const log = require("loglevel");
 
 const publish = (async (publicationName, routingKey, payload, resultHandler) => {
-    const broker = await Broker.create(config);
     try {
+        const broker = await Broker.create(config);

```

Comment:
I found it's possible that `Broker.create` would throw an error, and if I don't wrap the cod by `try/catch` then nothing will be thrown in the output, so it's hard to debug